### ISSUE TITLE
Error in README due to new version

### ISSUE THREAD

**Issue by Zorin95670** [erosb/json-sKema#66]
In Readme:
```
// create a validator instance for each validation (one-time use object)
Validator validator = Validator.create(schema, new ValidatorConfig(FormatValidationPolicy.ALWAYS));
```

But in version 0.12.0:
`new ValidatorConfig(FormatValidationPolicy.ALWAYS)` doesn't exists.

**Comment by erosb** [comment]
Hello @Zorin95670 , I released version 0.12.1 fixing the API compatibility, sorry about the inconvenience.

**Comment by Zorin95670** [comment]
Thank you for your support :smiley: 
Have a nice day!
